klyshko-operator/controllers/tuplegenerationtask_controller.go

```diff
@@ -250,7 +255,7 @@ func (r *TupleGenerationTaskReconciler) SetupWithManager(mgr ctrl.Manager) error
 
 // taskKeyFromName creates a RosterEntryKey from the given name and namespace. Expects that the zero-based VCP
 // identifier is appended with a hyphen to the name.
-func (r *TupleGenerationTaskReconciler) taskKeyFromName(namespace string, name string) (*RosterEntryKey, error) {
+func taskKeyFromName(namespace string, name string) (*RosterEntryKey, error) {
```

Comment:
Am I wrong that a receiver was not really needed here? If I am I will revert it and find another way.

Reply:
Not needed indeed. Removing is fine for me.